Hotfix preparedness number 1000

olethanh opened the pull request:
Explain what problem this PR is resolving

Related JIRA tickets : IA-XXX, WC2-XXX, POLIO-XXX

## Self proofreading checklist

- [ ] Did I use eslint and black formatters
- [ ] Is my code clear enough and well documented
- [ ] Are my typescript files well typed
- [ ] New translations have been added or updated if new strings have been introduced in the frontend
- [ ] My migrations file are included
- [ ] Are there enough tests
- [ ] Documentation has been included (for new feature)

## Changes

Explain the changes that were made. The idea is not to list exhaustively all the changes made (GitHub already provides a full diff), but to help the reviewers better understand:
- which specific file changes go together, e.g: when creating a table in the front-end, there usually is a config file that goes with it
- the reasoning behind some changes, e.g: deleted files because they are now redundant
- the behaviour to expect, e.g: tooltip has purple background color because the client likes it so, changed a key in the API response to be consistent with other endpoints

## How to test

Explain how to test your PR.
If a specific config is required explain it here (dataset, account, profile, ...)

## Print screen / video

Upload here print screens or videos showing the changes

## Notes

Things that the reviewers should know: known bugs that are out of the scope of the PR, other trade-offs that were made.
If the PR depends on a PR in bluesquare-components, or merges into another PR (i.o. main), it should also be mentioned here

beygorghor reviewed (approved):
This code is still complex but getting better ;-)
Maybe some unit test can be useful here one day...

quang-le commented on plugins/polio/js/src/hooks/useRfaTitle.ts:
```diff
@@ -0,0 +1,40 @@
+    const makeTitle = translateTitle(formatMessage);
+    const accessor = type === 'lqas' ? 'childabsent' : 'Tot_child_Absent_HH';
+    const childrenAbsentRound1 =
+        data && campaign && data[campaign]
```
I tried that 😅

quang-le commented on plugins/polio/js/src/hooks/useVerticalChartData.ts:
```diff
@@ -0,0 +1,42 @@
+    campaign?: string;
+    // eslint-disable-next-line no-unused-vars
+    formatter: (args: FormatForNFMArgs) => BarChartData[];
+    type: 'lqas' | 'im';
```
Not as straightforward as it seemed because `FormatForNFMArgs`was used elsewhere and relied on more restrictive typing. Solved it by using a generic type

quang-le commented on plugins/polio/js/src/utils/LqasIm.tsx:
```diff
@@ -98,3 +135,181 @@ export const makeAccordionData = ({
+
+    return [...convertedEntries, ...missingEntries].sort(sortGraphKeys);
+};
+export const verticalChartTooltipFormatter = (value, _name, props) => {
```
Boooo, lazy me

olethanh committed:
POLIO-1030 Change preparedness score formatting in email

olethanh committed:
Preparedness Fix for campaign CHD-2023-05-1_nOPV

the N'DJAMENA was escaped differently in different place and was making the code crash
